src/inspect.mjs

```diff
@@ -1,95 +1,489 @@
 import { surroundingAgent } from './engine.mjs';
 import { Type, Value, wellKnownSymbols } from './value.mjs';
 import {
-  Call, IsArray, Get, LengthOfArrayLike,
+  Assert,
+  Call,
   EscapeRegExpPattern,
+  Get,
+  HasOwnProperty,
+  IsAccessorDescriptor,
+  IsArray,
+  IsCallable,
+  IsDataDescriptor,
+  LengthOfArrayLike,
+  isArrayIndex,
+  isECMAScriptFunctionObject,
 } from './abstract-ops/all.mjs';
 import { Q, X } from './completion.mjs';
+import { OutOfRange } from './helpers.mjs';
 
 const bareKeyRe = /^[a-zA-Z_][a-zA-Z_0-9]*$/;
 
-const getObjectTag = (value, wrap) => {
-  let s;
-  try {
-    s = X(Get(value, wellKnownSymbols.toStringTag)).stringValue();
-  } catch {}
+const defaultOptions = {
+  colors: false,
+  showHidden: false,
+  showProxy: false,
+};
+Object.setPrototypeOf(defaultOptions, null);
+Object.seal(defaultOptions);
+
+const styles = {
+  bigint: 'yellow',
+  boolean: 'yellow',
+  date: 'magenta',
+  hidden: 'blackBright',
+  internalSlot: 'blackBright',
+  module: 'underline',
+  name: undefined,
+  null: 'bold',
+  number: 'yellow',
```

Comment:
can you make bigints and numbers blue? node doesn't use blue for them because it conflicts with the old powershell ui but we don't really have that concern.